inst/pages/19_visualization_techniques.qmd

```diff
@@ -276,41 +276,39 @@ at the Phylum level are available in the aforementioned chapter.
 ```{r}
 #| label = "plotAbundance1"
 # agglomerate tse by Order
-tse_order <- mergeFeaturesByRank(tse,
-                                rank = "Order",
-                                onRankOnly = TRUE)
+tse_phylum <- mergeFeaturesByRank(tse,
+                                  rank = "Phylum",
+                                  onRankOnly = TRUE)
+
 # transform counts into relative abundance
-tse_order <- transformAssay(tse_order,
-                              assay.type = "counts",
-                              method = "relabundance")
+tse_phylum <- transformAssay(tse_phylum,
+                             assay.type = "counts",
+                             method = "relabundance")
+
 # get top orders
-top_taxa <- getTopFeatures(tse_order,
-                       top = 10,
-                       assay.type = "relabundance")
+top_taxa <- getTopFeatures(tse_phylum,
+                           top = 5,
+                           assay.type = "relabundance")
+
 # leave only names for top 10 orders and label the rest with "Other"
-order_renamed <- lapply(rowData(tse_order)$Order,
-                   function(x){if (x %in% top_taxa) {x} else {"Other"}})
-rowData(tse_order)$Order <- as.character(order_renamed)
+rowData(tse_phylum)$Phylum[!rowData(tse_phylum)$Phylum %in% top_taxa] <- "Other"
```

Comment:
make two altExps, one for all phyla, one where these rare phyla are collapsed into "Other"? Then we still have access to all versions and the provenance is more clear.